fog/sql_recovery_db/src/bin/fog_sql_recovery_db_migrations.rs

```diff
@@ -8,16 +8,26 @@ extern crate diesel_migrations;
 
 use diesel::{prelude::*, PgConnection};
 use diesel_migrations::embed_migrations;
-use std::env;
+use std::{
+    env,
+    io::{stdout, Write},
+};
 
 embed_migrations!("migrations/");
 
 fn main() {
     let database_url = env::var("DATABASE_URL").expect("Missing DATABASE_URL environment variable");
+    println!("Got DATABASE_URL={}", database_url);
```

Comment:
I think it should go away, the database url contains secrets and there's no need for that to end in k8 logs.